packages/components/src/tripInput/TripInput.js

```diff
@@ -22,6 +22,10 @@ type Props = {|
   +placeholder?: string,
 |};
 
+const getEmptyLocationPlaceholder = (field: string): string => {
```

Comment:
What about `getAnywherePlaceholder`? Just an idea

Reply:
Would also be nice! but in the case of `From` it's not really anywhere... hahaha, we can change it later!